lib/repo/subversion_repository.rb

```diff
@@ -584,25 +579,23 @@ def self.__read_in_authz_file
       return file_content
     end
 
-    # Semi-private class method: Writes out Repository.conf[:REPOSITORY_PERMISSION_FILE]
+    # Semi-private class method
     def self.__write_out_authz_file(authz_file_contents)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
-        raise ConfigurationError.new("Required config 'IS_REPOSITORY_ADMIN' not set")
+  
+      if !MarkusConfigurator.markus_config_repository_admin?
+        raise NotAuthorityError.new("Unable to write authsz file:  Not in authoritative mode!");
```

Comment:
Line is too long. [96/80]<br>Do not use semicolons to terminate expressions.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.